packages/curve-ui-kit/src/entities/themes/lib/generate-themes.ts

```diff
@@ -0,0 +1,19 @@
+import { createTheme as createMuiTheme, type Theme } from '@mui/material/styles'
+import { basicMuiTheme, type ThemeKey } from '@/shared/lib/basic-theme'
```

Comment:
`'@shared/lib'` should map to [curve-lib/src/shared/lib](https://github.com/curvefi/curve-frontend/tree/b6327360c6d88e7f7b715977011f48ce10e28470/packages/curve-lib/src/shared/lib)!

Reply:
The only imports from this pack will be
`entities/themes`
`shared/ui`